openedx/core/lib/api/tests/mixins.py

```diff
@@ -0,0 +1,53 @@
+"""
+Mixins for API Authentication tests.
+"""
+from time import time
+
+from django.conf import settings
+import jwt
+
+
+JWT_AUTH = 'JWT_AUTH'
+
+
+class JwtMixin(object):
+    """ Mixin with JWT-related helper functions. """
+
+    JWT_SECRET_KEY = getattr(settings, JWT_AUTH)['JWT_SECRET_KEY']
+    JWT_ISSUER = getattr(settings, JWT_AUTH)['JWT_ISSUER']
+    JWT_AUDIENCE = getattr(settings, JWT_AUTH)['JWT_AUDIENCE']
```

Comment:
I think you will not be able to use `@override_settings` if you set up these values as class variables.  I would suggest retrieving their values in a `setUp` method instead.